src/core/Group.cpp

```diff
@@ -140,35 +144,28 @@ QImage Group::icon() const
     }
 }
 
-QPixmap Group::iconPixmap() const
+QPixmap Group::iconPixmap(int size) const
```

Comment:
IMHO, these functions should also explicitly take the enum, not an int. Discourages inconsistent icon sizes and you can always add a size to the enum itself.

Reply:
Oh good point now that we have enums

Reply:
done